lib/cmd/flags.go

```diff
@@ -88,3 +92,16 @@ func redact(flag, val string) string {
 
 	return val
 }
+
+// maybeRedactToken redacts the last element of the given path
+// if it is a hex token of more than 16 chars.
```

Comment:
Technically 15...

Reply:
Should we add more context like "to remove secrets"? I'd never guess why we do this.

Reply:
see:
```
// redact returns a redacted version of the given flag value. It currently supports redacting
// passwords in valid URLs as well as flags that contains words like "token", "password", "secret", "db" or "key".
func redact(flag, val string) string {
```